.circleci/config.yml

```
@@ -410,10 +410,14 @@ workflows:
     jobs:
       - build:
           filters:
+            branches:
+              ignore: /tests.*/
```

```suggestion
              ignore: /^tests\//
```
might be safer? right now it'll match any branch with `tests` in the name

---

Actually, the `^` is not needed here. The current regex matches branches that start with `tests` (I've tested the behaviour on CircleCI).

I agree that adding `\/` makes sense. I'll do that now.

---

Updated now.

I have verified that branches containing `tests/` not at the start are not ignored by this.

---

interesting! Thanks for testing.